components/date-picker/date-picker.jsx

```diff
@@ -85,6 +85,10 @@ const propTypes = {
 	 * Value of input that gets passed to `parser` prop on initial render. This prop is only present for uncontrolled use of Datepicker which is _highly discouraged_. A better name for this prop would be `defaultFormatedValue`. Please use the `value` prop instead. _Not tested._
 	 */
 	formattedValue: PropTypes.string,
+	/**
+	 * Applies the error style to the component.
+	 */
+	hasError: PropTypes.bool,
```

Comment:
is `hasError` a required prop?

Reply:
No, it's optional and will be interpreted as false when not passed in.